Source/Atj/data/mvp_scenario.json

```diff
@@ -446,6 +494,81 @@
                     "target": "meeting_room_chair_1"
                 }
             ]
+        },
+        {
+            "name": "debug_player_move_to_refrigerator_routine",
+            "tasks": [
+                {
+                    "type": "behavior",
+                    "sync_time": "0.0",
+                    "behavior": "move_to",
+                    "target": "refrigerator_1"
+                },
+                {
+                    "type": "execute_action",
+                    "sync_time": "20.0",
+                    "action": "bind_player_to_debug_player_move_to_storage_cabinet_1_routine_action"
+                }
+            ]
+        },
+        {
+            "name": "player_move_to_storage_cabinet_1_routine",
+            "tasks": [
+                {
+                    "type": "behavior",
+                    "sync_time": "0.0",
+                    "behavior": "move_to",
+                    "target": "storage_cabinet_1"
+                },
+                {
+                    "type": "execute_action",
+                    "sync_time": "20.0",
+                    "action": "bind_player_to_debug_player_move_to_zaks_desk_routine_action"
```

Comment:
Should this be changed to (stars excluded):
`bind_player_to_debug_player_move_to_zaks_desk**_1_**routine_action`?

Reply:
This still needs to be updated @Slohive